buildtest/menu/build.py

```diff
@@ -143,28 +146,33 @@ def func_build_subcmd(args):
 
     check_configuration(config_file)
 
-    # Discover list of one or more config files based on path provided
-    config_files = discover_configs(args.config)
+    # Discover list of one or more buildspec files based on path provided
+    buildspecs = discover_buildspecs(args.buildspec)
 
     # if no files discovered let's stop now
-    if not config_files:
+    if not buildspecs:
         msg = "There are no config files to process."
         sys.exit(msg)
 
-    print("\n {:^45} \n".format("Discovered Files"))
-    [print(config) for config in config_files]
-    print("\n\n")
-
     logger.debug(
-        f"Based on input argument: -c {args.config} buildtest discovered the following configuration {config_files}"
+        f"Based on input argument: --buildspec {args.buildspec} buildtest discovered the following Buildspecs: {buildspecs}"
     )
 
     if args.exclude:
         logger.debug(f"The exclude config pattern are the following: -e {args.exclude}")
-        config_files = [
-            config for config in config_files if include_file(config, args.exclude)
+        buildspecs = [
```

Comment:
If you use buildspecs here, I think we should be consistent and use the same variable in the previous function instead of buildspec_files.

Reply:
okay i this fix this locally and call them ``buildspecs`` instead of ``buildspec_files``  this makes sense.